cognite_toolkit/_cdf_tk/apps/_dump_app.py

```diff
@@ -50,6 +51,10 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             self.command("group")(DumpConfigApp.dump_group)
             self.command("node")(DumpConfigApp.dump_node)
 
+            if Flags.DUMP_EXTENDED.is_enabled():
+                self.command("location-filter")(DumpConfigApp.dump_location_filters)
```

Comment:
This was forgotten here, so adding it here instead of a separate PR. Please forgive me for adding an extra line to this PR not directly related.